etc/profile-m-z/skypeforlinux.profile

```diff
@@ -15,15 +15,22 @@ ignore nou2f
 ignore novideo
 ignore private-dev
 ignore dbus-user none
```

Comment:
Move this (`ignore dbus-user none`) out of the "Disabled until someone reported positive feedback" block.